#!/usr/bin/env python
PACKAGE = "ohm_neg_obst"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("map_height", double_t, 0, "Adjust the size of the map", 20.0, 1.0, 100.0) 
gen.add("map_width" , double_t, 0, "Adjust the size of the map", 20.0, 1.0, 100.0) 
gen.add("map_res", double_t, 0, "Adjust the resolution of the map", 0.05, 0.01, 0.1)     
gen.add("growing", bool_t, 0, "Flip to use growing", True)    
gen.add("n_grow", int_t, 0, "Adjust the layer of cells for the growing", 1, 1, 10)      
gen.add("thresh_empty", int_t, 0, "Adjust the number of empty neighbours of a cell to consider it a wall", 1, 1, 8)



exit(gen.generate(PACKAGE, "ohm_neg_obst", "ReconfigureHeightMapper"))